src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -353,6 +353,26 @@ export default defineConfig({
 });
 ```
 
+### `skewProtection`
+
+**Type:** `boolean`<br/>
+**Available for:** Serverless
+
+Use this property to enable [Vercel Skew protection](https://vercel.com/docs/deployments/skew-protection). Reminder that Skew protection is only available for Pro and Enterprise accounts.
```

Comment:
```suggestion
Use this property to enable [Vercel Skew protection](https://vercel.com/docs/deployments/skew-protection) (available with Vercel Pro and Enterprise accounts).
```
Fun pedantic language nit: it wouldn't be a "reminder" if they never knew in the first place!

Also, wanted to make it clear these are VERCEL accounts, not possibly interpreted as needing an *Astro* Pro account. Juuuust in case!